languages/wikipron/src/scrape_and_write.py

```diff
@@ -0,0 +1,123 @@
+import wikipron
+from readme_insert import readme_insert
+from datetime import datetime
+from time import sleep
+import requests
+import os
+import json
+
+
+def call_scrape(lang, config, file_extension):
```

Comment:
`file_extension` -> `filename_suffix` or similar? (The term file extension makes me think of `.tsv`, `.py`, etc. instead.)